tests/cbmc/proofs/s2n_stuffer_printf/s2n_stuffer_printf_harness.c

```diff
@@ -34,7 +45,10 @@ void s2n_stuffer_printf_harness()
     /* CBMC defines va_list as void** */
     size_t va_list_size;
     __CPROVER_assume(va_list_size % sizeof(void*) == 0);
+    __CPROVER_assume(va_list_size > 0);
     void** va_list_mem = malloc(va_list_size);
+    __CPROVER_assume(va_list_mem != NULL);
+    va_list_mem[va_list_size / sizeof(void*) - 1] = NULL;
 
```

Comment:
Why does the last pointer need to be NULL?

Reply:
My bad, we can really only rely on the object size, no terminator here. Fixed.